src/components/HorizontalScores.tsx

```diff
@@ -0,0 +1,24 @@
+interface Scores {
+  rmpScore: number;
+  diffScore: number;
+  wtaPercent: number;
+}
+
+export const HorizontalScores = ({ rmpScore, diffScore, wtaPercent }:Scores) => {
+  const getScoreColor = (value: number, outOf: number) => {
+    value /= outOf;
+    const hue = (value * 120).toString(10);
+    return ["hsl(", hue, ",100%,85%)"].join("");
+  }
+
+  return (
+    <div className="grid grid-cols-3">
+      <h2 className="bg-blue-dark text-white text-center py-1 rounded-tl-2xl">RMP</h2>
```

Comment:
Pull the repeated tailwindcss out of here and put it in a const variable. Then, use concatenation to add the extra in! It'll be much neater